parent/pom.xml

```diff
@@ -467,6 +468,53 @@
             </build>
         </profile>
 
+        <!--
+            Profile that activates automatically when maven.test.skip is set to true and excludes the test phase
+        -->
+        <profile>
+            <id>skip-all-tests</id>
+            <activation>
+                <property>
+                    <name>maven.test.skip</name>
+                    <value>true</value>
```

Comment:
Value is unnecessary, because with use `-Dmaven.test.skip` notation in command line calls.

Reply:
I'd prefer to keep this explicit as it is so in maven's documentation